### PR TITLE
Expand description

### PR DESCRIPTION
**Pull request by phadej**
cc @RyanGlScott

### CODE REVIEW

File: with-index-instances/with-index-instances.cabal [hunk 1]
```diff
@@ -7,9 +7,15 @@ license-file:  LICENSE
 category:      Data
 maintainer:    Oleg Grenrus <oleg.grenrus@iki.fi>
 author:        Edward Kmett
-synopsis:      FunctorWithIndex, FoldableWithIndex, TraversableWithIndex
+synopsis:      More instances of FunctorWithIndex, FoldableWithIndex, TraversableWithIndex
 description:
-  This package provides extra instances for type-classes in @with-index@
+  This package provides extra instances for type-classes in @with-index@ package.
```
Comment (RyanGlScott) [inline with-index-instances/with-index-instances.cabal:12]:
```suggestion
  This package provides extra instances for type-classes in the @with-index@ package.
```

File: with-index-instances/with-index-instances.cabal [hunk 2]
```diff
@@ -7,9 +7,15 @@ license-file:  LICENSE
...
+  .
+  The intention is to keep this package minimal,
```
Comment (RyanGlScott) [inline with-index-instances/with-index-instances.cabal:14]:
```suggestion
  The intention is to keep this package minimal;
```

File: with-index-instances/with-index-instances.cabal [hunk 3]
```diff
@@ -7,9 +7,15 @@ license-file:  LICENSE
...
+  it provides instances formely existed in @lens@ or @optics-extra@.
```
Comment (RyanGlScott) [inline with-index-instances/with-index-instances.cabal:15]:
```suggestion
  it provides instances that formerly existed in @lens@ or @optics-extra@.
```

File: with-index-instances/with-index-instances.cabal [hunk 4]
```diff
@@ -7,9 +7,15 @@ license-file:  LICENSE
...
+  We recommend to put other instances directly into their defining packages.
```
Comment (RyanGlScott) [inline with-index-instances/with-index-instances.cabal:16]:
```suggestion
  We recommend putting other instances directly into their defining packages.
```

**Review by RyanGlScott — COMMENTED** [review]
Aside from some minor grammar-relation suggestions, LGTM.

**Comment by phadej** [comment]
@RyanGlScott, thanks for corrections!

**Pull request merged**